hera_mc/correlator_levels.py

```diff
@@ -18,7 +18,7 @@
 default_levels_filename = 'levels.tmp'
 
 
-def get_levels(pf_input, testing, network='local', timeout=5):
+def get_levels(pf_input, testing, network='local', timeout=2):
```

Comment:
The change of the timeout here is an example of an unrelated change that should really go in its own commit, so that the commit message can explain the change. Unless it's not unrelated?

(If Future Me wanted to know how this timeout was chosen, I'd use `git blame` and then `git log` to find this commit, and then I'd be stuck since there is presently no explanation for why the change was made, and it is not at all obvious to me what the change has to do with the general thrust of this pull request.)

Reply:
I was just getting impatient with waiting the 5 seconds...